app/assets/javascripts/admin/addon/routes/admin-logs-staff-action-logs.js

```diff
@@ -7,6 +7,14 @@ export default DiscourseRoute.extend({
     filters: { refreshModel: true },
   },
 
+  beforeModel(transition) {
+    const params = transition.to.queryParams;
+    const controller = this.controllerFor("admin-logs-staff-action-logs");
+    if (controller.filters === null || params.force_refresh) {
```

Comment:
Could you avoid this code by adding `refreshModel: true` above for `force_refresh`?

Reply:
I just tried doing that and it's not refreshing the content based on filter. Note that the query params `filters` is also changing and it already has `filters: { refreshModel: true }` but that is also not refreshing the content.

Reply:
OK cool let's merge it.